src/app/hooks/useViewTracker/index.test.tsx

```diff
@@ -646,7 +646,7 @@ describe('useViewTracker', () => {
       expect(options).toEqual({ threshold: [0.5] });
       expect(mockOptimizelyTrack).toHaveBeenCalledTimes(1);
       expect(mockOptimizelyTrack).toHaveBeenCalledWith(
-        'component_views',
+        'component-views',
```

Comment:
As above, I thought this was no longer used. Can we check? Does this mean there's a test we can delete?